### ISSUE TITLE
Release to remove warnings

### ISSUE THREAD

**Issue by alexlafroscia** [CargoSense/briefly#8]
Hey -- I noticed that #7 was merged to remove the warnings about missing parenthesis. Could you cut a new Hex release so that people can benefit from that improvement?

Thanks!

**Comment by lpil** [comment]
Can we get this actioned please @benwilson512 ? :)

**Comment by joevandyk** [comment]
@cargosensedev @benwilson512 any chance on this one?

Seeing a ton of these

```
warning: variable "package" does not exist and is being expanded to "package()", please use parentheses to remove the ambiguity or change the variable name
  /Users/joe/projects/print_server_v2/deps/briefly/mix.exs:11: Briefly.Mixfile.project/0

warning: variable "deps" does not exist and is being expanded to "deps()", please use parentheses to remove the ambiguity or change the variable name
  /Users/joe/projects/print_server_v2/deps/briefly/mix.exs:12: Briefly.Mixfile.project/0

warning: variable "default_env" does not exist and is being expanded to "default_env()", please use parentheses to remove the ambiguity or change the variable name
  /Users/joe/projects/print_server_v2/deps/briefly/mix.exs:21: Briefly.Mixfile.application/0
```

**Comment by TheFirstAvenger** [comment]
+1 to a release please 😄

**Comment by stocks29** [comment]
A hex release for this would be greatly appreciated :)

**Comment by sgerrand** [comment]
👋 @benwilson512, are you able to release a new version of this package to Hex? It's been a long time since the last one.

**Comment by mcrumm** [comment]
I just released v0.4.0 on hex :) https://hex.pm/packages/briefly/0.4.0
